src/sentry/api/endpoints/project_releases.py

```diff
@@ -98,7 +98,7 @@ def get(self, request, project):
             queryset=queryset,
             order_by='-sort',
             paginator_cls=OffsetPaginator,
-            on_results=lambda x: serialize(x, request.user),
+            on_results=lambda x: serialize(x, request.user, project=project),
```

Comment:
so the way we did this in the past was to explicitly pass the serializer class with an arg on instantiation. not against this change though